test/integration/test_utils.js

```diff
@@ -203,6 +214,7 @@ async function dragAndDropAnnotation(page, startX, startY, tX, tY) {
   await page.waitForTimeout(10);
   await page.mouse.move(startX + tX, startY + tY);
   await page.mouse.up();
+  await page.waitForTimeout(10);
```

Comment:
In recent intermittent-fixes done by Tim the use of this sort of *very short* timeout was identified as a potential issue on slower hardware.
Any chance that we could await *something* rather than relying on a timeout here?